packages/desktop-client/src/components/transactions/MobileTransaction.js

```diff
@@ -1043,7 +1040,9 @@ export class TransactionList extends Component {
         }
 
         sections.push({
-          id: transaction.date,
+          id: `${isPreviewId(transaction.id) ? 'preview/' : ''}${
+            transaction.date
+          }`,
```

Comment:
`id` is used as a `key`. And sometimes the schedule date clashes with transaction date. Thus creating two entries in the list with the same `id`. Thus causing all sorts of weird issues.